timemachine/fe/atom_mapping.py

```diff
@@ -333,6 +333,21 @@ def _deduplicate_all_cores_and_bonds(all_cores, all_bonds):
     return cores, bonds
 
 
+def cb_count(mol_a, mol_b, core):
```

Comment:
nit: `core_bounds_broken_count` or something more explicit, since `cb` could be core bonds.